### PR TITLE
Initial security policy update

### PR DESCRIPTION
**Pull request by JKarlavige**
## Description & motivation

To improve the mozilla security score for Docs

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [x] No: please ensure the base branch is `current`

### CODE REVIEW

**Comment by JKarlavige** [comment]
Currently scoring a B with the latest preview tests [here](https://observatory.mozilla.org/analyze/deploy-preview-837--docs-getdbt-com.netlify.app)

File: website/Makefile [hunk 1]
```diff
@@ -19,6 +19,8 @@ build:
 	cat ../_redirects | grep 'docs/' | awk '{ print "/v0.11" $$1 "\t" $$2 "\t" $$3 }' >> build/_redirects
 	cat ../_redirects | grep 'docs/' | awk '{ print "/v0.10" $$1 "\t" $$2 "\t" $$3 }' >> build/_redirects
 
+	cat ../_headers > build/_headers
```
Comment (lfracalossiFA) [inline website/Makefile:22]: Just for the sake of my own curiosity, why did we need to do this? Is it just outputting the _headers file to the directory Netlify expects it to be in?

File: _headers [hunk 2]
```diff
@@ -1,2 +1,5 @@
 /*
   Strict-Transport-Security: max-age=63072000; includeSubDomains; preload
+  Content-Security-Policy: object-src 'none'; frame-ancestors 'none';
```
Comment (lfracalossiFA) [inline _headers:3]: I see you, too, decided that a more restrictive CSP wasn't something you wanted to mess with :)

**Review by lfracalossiFA — APPROVED** [review]

File: website/Makefile (hunk 1 above)
Comment (JKarlavige) [reply website/Makefile:22]: After updating the headers file and not seeing changes, I noticed on the Netlify deploy page it said "no headers processed". The headers file was not being included in the final build. This line adds the headers file to the build folder, similar to how jekyll adds it to the _site folder.

File: _headers (hunk 2 above)
Comment (JKarlavige) [reply _headers:3]: @lfracalossiFA You are absolutely right, a lot of potential for things to go wrong with a strict policy

**Pull request merged**